php/helpers.php

```diff
@@ -25,13 +25,27 @@ function block_field( $name, $echo = true ) {
 	 */
 	global $block_lab_attributes, $block_lab_config;
 
+	$default_fields = array( 'className' );
+
 	if ( ! isset( $block_lab_attributes ) || ! is_array( $block_lab_attributes ) ) {
 		return null;
 	}
 
-	$default_fields = array( 'className' );
+	$is_name_allowed = isset( $block_lab_config->fields[ $name ] ) || in_array( $name, $default_fields, true );
+
+	/**
```

Comment:
Should this be a single asterix, since it's an inline comment, not a doc block?

i.e.

```
/*
 *
 *
 */
```

Reply:
Ah, good question. 

The examples I've seen [in Core](https://github.com/WordPress/wordpress-develop/blob/00b03f2a6f2c21ebf9975c9c08976358681400ba/src/wp-includes/feed.php#L28) have had 2 asterisks, but I'm not sure if there's an official convention for this.